src/common/wifi/core/CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ target_sources(wifi-core
 target_link_libraries(wifi-core
     PUBLIC
         notstd
+        ${PROJECT_NAME}-protocol
```

Comment:
Need to remove this to eliminate upward dependency, see related comments elsewhere.